### ISSUE TITLE
Image fails to load in reddit with Reddit Enhancement Suite

### ISSUE THREAD

**Issue by lpaulger** [openseadragon/reddit-bot#3]
I like the idea of zoomable images but the first impression I get is the image below.  I'm sure there would be something else that would be more useful to display there.
<img width="842" alt="screen shot 2016-06-05 at 11 17 29" src="https://cloud.githubusercontent.com/assets/1000544/15804397/54507258-2b0f-11e6-875c-775ac354585f.png">

**Comment by avandecreme** [comment]
Thanks for reporting.
It looks like you are using Reddit Enhancement Suite, right?
If so, I think RES believes the link is an image and tries to display it directly.

**Comment by lpaulger** [comment]
yup, I guess the take away is this is how any user with RES will see the link.

**Comment by avandecreme** [comment]
Ok, I will see with the RES developers how we can fix this.

**Comment by iangilman** [comment]
Would be awesome if RES could add OpenSeadragon support!

**Comment by harrisongill** [comment]
I'm not sure if it is using a different browser, but I do not even get the loading link. Luckily, the image does open if I click the link however. This is using RES in Safari.

<img width="879" alt="screen shot 2016-08-27 at 7 05 00 pm" src="https://cloud.githubusercontent.com/assets/4271517/18031179/51122d64-6c89-11e6-8da2-586a60d45d8c.png">

**Comment by avandecreme** [comment]
So according to the RES developers, it is fixed in their beta version.
